CONTRIBUTING.md

```diff
@@ -1,118 +1,182 @@
-   alternatively `flit install --symlink` (this means that
-   the installed code will change as soon as you change it in the
-   download location).
-7. Run `tox -e reformat` to make the format consistent.
-8. Run `tox -e check` to check all the formatting is right.
-   (The reformat command will not deal with un-used imports)
-9. Create a new branch.
-10. Make changes and commit.
-11. [Push to your
-    fork](https://help.github.com/articles/pushing-to-a-remote/).
-12. [Open a pull
-    request!](https://help.github.com/articles/creating-a-pull-request/)
-
-For more detailed explanations, please check out the [Astropy
-development
-docs](http://docs.astropy.org/en/stable/development/workflow/development_workflow.html).
+4. [Clone your fork](https://help.github.com/articles/cloning-a-repository/)
+   (remote name will be `origin`)
+5. Add an `upstream` remote with `git remote add upstream https://github.com/poliastro/poliastro.git`
+   and fetch its information with `git fetch upstream`
+6. Set your `main` branch to track `upstream` using `git branch --set-upstream-to=upstream/main`
+
+Next, configure your Python environment:
+
+6. Install Python for development.
+7. Create a Python virtual environment using `python -m venv .venv`
+8. Activate it using `source .venv/bin/activate`
+9. Upgrade the development dependencies using `python pip install -U pip setuptools wheel flit tox`
+10. Install the code in development mode using `flit install --simlink`
+    (this means that the installed code will change as soon as you change it in the
+    download location).
+
+And with this, you will be ready to start contributing!
+
+### Pull request workflow
+
+Every time you want to contribute some code or documentation to poliastro
+you will need follow these steps:
+
+1. Make sure that your `main` branch is up to date: `git switch main`
+   (or `git checkout main`) and `git pull --rebase upstream main`
+2. Create a new branch using `git switch --create descriptive-name-of-my-change`.
+3. Make your changes!
+4. Commit your changes using `git commit -m 'Add new awesome feature'`.
+5. [Push to your fork](https://help.github.com/articles/pushing-to-a-remote/).
+6. [Open a pull request!](https://help.github.com/articles/creating-a-pull-request/)
+
+One branch corresponds to one pull request. This means that,
+if you keep committing and pushing changes to the same branch,
+the pull request will update automatically (you don't need to open a new one!).
+
+When your pull request is merged, you can:
+
+7. Update your `main` branch again: `git switch main` and `git pull --rebase upstream main`
+8. Delete your local branch: `git branch --delete descriptive-name-of-my-change`
+9. Refresh your fork: `git fetch --prune origin` and `git push origin main`
+
+Remember that, whenever you want to start a new pull request,
+you need to start from step 1.
+
+### Build the documentation
+
+To build the docs, run:
+
+```console
+(.venv) $ cd docs
```

Comment:
Was the inclusion of `(.venv)` intentional?

Reply:
Yes! It's because the virtual environment is activated